docs/JLBP-1.md

```diff
@@ -82,11 +83,5 @@ In Gradle these libraries should have `compileOnly` scope:
 
 The `provided` scope should be used when the dependency is needed at runtime
 and compile time. However the specific jar will be supplied by the environment
-in which the code runs; for example a Java servlet container.
-
-
-
-- Use the smallest scope possible. For example, AutoValue doesn't
```

Comment:
I know that you use AutoValue in your examples, but I think it is still worth calling it out in the text because it's a common mistake in Google-owned Java libraries.

Reply:
done